### DISCUSSION TITLE
How can I see the reason for why events are getting dropped?

### DISCUSSION THREAD

**Discussion by sbsce** [getsentry/support#52] [Support] [answered]
Very simple question, how can I see the reason for why events are getting dropped on Sentry? I am sure there is a way to see that?

I am seeing that all my errors are counting as "Dropped" in the past 2 days, but I can't find anywhere why they are being dropped. On the "stats" page it just says "Dropped", with no further explanation.

**Comment by Firefly74940** [comment]
Same Issue here with an UE4 project using default crash reporter, did not change anything

**Reply by sbsce** [reply]
I am also using UE4 with the default crash reporter. So it seems there is a general issue that breaks UE4 crash reporting on Sentry then.

**Reply by Firefly74940** [reply]
i think the related issue to track would be this one now https://github.com/getsentry/sentry/issues/37106

But anyway having some info on why events are dropped would be a good thing

**Comment by danielkhan** [comment]
We are currently discussing surfacing dropped events + the reason.

**Comment by souredoutlook** [comment, marked as answer]
It is possible to look at your organization's stats grouped by the project, outcome and reason dimensions [via the API](https://docs.sentry.io/api/organizations/retrieve-event-counts-for-an-organization-v2/). This would provide a bit more detail than the current stats page.

As @danielkhan mentioned it's not currently a feature of the app to show both the outcome and reason on the stats page. 

TY @Firefly74940 for pointing to the root cause in this case. 😄
